src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -934,14 +965,20 @@ def new_chunk():
     return full_text, optional
 
 
-def _find_longest_common_sequence(sequences):
+def _find_longest_common_sequence(sequences, token_timestamp_sequences=None):
```

Comment:
could you add a line of doc about the new argument (is not entirely straightforward!)